ui/src/singletons/store.ts

```diff
@@ -171,6 +177,7 @@ const useStore = create<AppStore>()(
             edges: [],
             eipNodeConfigs: {},
             selectedChildNode: null,
+            layout: "horizontal"
```

Comment:
I think it makes sense to preserve the current layout, even if the flow is cleared.